{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/templates/admin/base_site.html

```diff
@@ -0,0 +1,11 @@
+{% raw %}{% extends "admin/base.html" %}
+
+{% block extrastyle %}{% endblock %}
+{% block extrahead %}{% endblock %}
+{% block title %}{{ title }} | {% endraw %}{{ cookiecutter.project_name }}{% raw %}{% endblock %}
```

Comment:
`{{ cookiecutter.project_name }}` should be enclosed with double quotes for django templates to work here.